table/server/common/src/main/java/alluxio/table/common/transform/action/WriteAction.java

```diff
@@ -26,8 +27,10 @@
  */
 public class WriteAction implements TransformAction {
   private static final String NAME = "write";
-  private static final String NUM_FILES_OPTION = "hive.num.files";
-  private static final int DEFAULT_NUM_FILES = 1;
+  private static final String NUM_FILES_OPTION = "hive.file.maxcount";
+  private static final String FILE_SIZE_OPTION = "hive.file.minsize";
```

Comment:
I think our convention is like:
```suggestion
  private static final String FILE_SIZE_OPTION = "hive.file.size.min";
```